addons/sale_project/views/project_task_views.xml

```diff
@@ -109,6 +132,12 @@
             <field name="partner_id" position="after">
                 <field name="sale_order_id" filter_domain="['|', ('sale_order_id', 'ilike', self), ('sale_line_id', 'ilike', self)]"/>
             </field>
+            <field name="partner_id" position="attributes">
+                <attribute name="invisible">context.get('should_not_display_partner')</attribute>
```

Comment:
Can you rename the context key? Like this: `display_partner` and the value should be the opposite of course 😄 . IMO, I think it is better to have a positive key/variable instead of having a negative key/variable name. 🙂

Reply:
Then we have to add `not` before context and context default value should be True like this `not context.get('display_partner', True)`. Is it ok? :thinking:

Reply:
Indeed good idea thanks @auon-odoo